src/Microsoft.OData.Client/Serialization/HttpClientRequestMessage.cs

```diff
@@ -40,17 +41,18 @@ public class HttpClientRequestMessage : DataServiceClientRequestMessage, ISendin
             "Last-Modified",
         }, StringComparer.OrdinalIgnoreCase);
 
-        /// <summary>Request Url.</summary>
-        private readonly Uri _requestUrl;
-
         /// <summary> The effective HTTP method. </summary>
         private readonly string _effectiveHttpMethod;
 
         /// <summary>HttpRequestMessage instance.</summary>
         private readonly HttpRequestMessage _requestMessage;
         private readonly HttpClient _client;
-        private readonly HttpClientHandler _handler;
         private readonly MemoryStream _messageStream;
+        private readonly bool _shouldDisposeClient;

```

Comment:
In what instance would we use this, I think with HttpClient we want to always keep it around for the life of the application.

Reply:
Good call out. I no longer need this now that the default client is static.